src/form/Form.jsx

```diff
@@ -516,6 +516,10 @@ function Form({
       return errors;
     };
 
+    initState.setValuesCallbacks[formKey] = () => {
+      setRender((render) => !render);
```

Comment:
should we autogenerate a uuid unique to this form in a state variable? and case on that rather than the formKey since potentially they might have the same form rendered in multiple places (e.g. our own job app form)

also this could be moved into a useEffect with an empty dependency array since this function doesn't need to be refreshed every time a new step is loaded

in the cleanup function of that useEffect you could remove both the validate and render callbacks since we don't want to attempt to call the functions of an unmounted component